patches/minecraft/com/mojang/blaze3d/vertex/BufferBuilder.java.patch

```diff
@@ -1,14 +1,14 @@
 --- a/com/mojang/blaze3d/vertex/BufferBuilder.java
 +++ b/com/mojang/blaze3d/vertex/BufferBuilder.java
-@@ -486,4 +_,16 @@
+@@ -486,4 +_,17 @@
           this.f_166822_ = p_166829_;
        }
     }
 +
 +   // Forge start
 +   public void putBulkData(ByteBuffer buffer) {
 +      m_85722_(buffer.limit() + this.f_85658_.m_86020_());
-+      this.f_85648_.position(this.f_85654_ * this.f_85658_.m_86020_());
++      this.f_85648_.position(this.f_85652_);
```

Comment:
This should be the next element byte, so `f_85654_`

Reply:
`f_85652_` is `nextElementByte`. `f_85654_` is `vertices`.